assets/icons.tsx

```diff
@@ -132,6 +132,25 @@ export const ArrowLeftCircleIcon = () => {
   );
 };
 
+export const ArrowLeftIcon = () => {
+  return (
+    <svg
+      xmlns='http://www.w3.org/2000/svg'
+      width='21'
+      height='20'
+      viewBox='0 0 21 20'
+      fill='none'>
+      <path
+        d='M16.3332 9.99996H4.6665M4.6665 9.99996L10.4998 15.8333M4.6665 9.99996L10.4998 4.16663'
+        stroke='#131212'
+        stroke-width='1.66667'
```

Comment:
React supports `strokeWidth` for `stroke-width`. Basically you have to discard the hypen from attribute name and the first character of second word will be in uppercase. e.g. `strokeLinecap`, `strokeLinejoin`.